curriculum/get-challenges.js

```diff
@@ -293,7 +292,7 @@ Challenges that have been already audited cannot fall back to their English vers
       ([id]) => id === challenge.id
     );
 
-    challenge.block = meta.name ? dasherize(meta.name) : null;
+    challenge.block = meta.dashedName ? meta.dashedName : null;
```

Comment:
This is what made me think we don't need both. We could do something like

```suggestion
    if (!meta.dashedName)
      throw Error(`challenge ${challenge.title}'s meta.json has no dashedName`);

    challenge.block = meta.dashedName;
```

Reply:
I looked into this a bit more and it turns out we do validate block like this:

https://github.com/freeCodeCamp/freeCodeCamp/blob/2f9ce8d7a5edea7dc5fb17cb968ee69e1a25f13c/curriculum/schema/challenge-schema.js#L26

so we don't need extra validation.

I would just drop the ternary and do `challenge.block = meta.dashedName`